modules/ai/assets/js/editor/components/prompt-credits.js

```diff
@@ -11,7 +11,7 @@ const PromptCredits = ( props ) => {
 		<Typography variant="caption" color="text.tertiary">
 			{
 				/* Translators: %s: AI prompt remained credits. */
-				sprintf( __( 'You\'ve used %s of your Elementor AI plan.', 'elementor' ), props.usagePercentage + '%' )
+				sprintf( __( 'You\'ve used %s credits for this AI feature.', 'elementor' ), props.usagePercentage + '%' )
```

Comment:
Backtick?

Reply:
I didn't change it in the original [commit](https://github.com/elementor/elementor/commit/f822f03e04c06c1140c80dc8b6a9ebc034d2bb4c), that's how it was written